pkg/common/config/config.go

```diff
@@ -382,23 +382,23 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 
 		if vcConfig.User == "" {
 			vcConfig.User = cfg.Global.User
-			if vcConfig.User == "" {
+			if vcConfig.User == "" && vcConfig.VCSessionManagerURL == "" {
```

Comment:
Do we need to add separate validation about `vcConfig.VCSessionManagerURL`?
if user is empty and VCSessionManagerURL is empty, we are printing `log.Errorf("vcConfig.User is empty for vc %s!", vcServer)`  I think goal is validate either  one of it is set.